src/Swashbuckle.AspNetCore.SwaggerGen/DependencyInjection/SwaggerGenJsonOptions.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System.Text.Json;
+
+namespace Swashbuckle.AspNetCore.SwaggerGen.DependencyInjection;
+
+public class SwaggerGenJsonOptions
+{
+    public JsonSerializerOptions SerializerOptions { get; set; }
+}
```

Comment:
Could you add XML documentation to this type and the property explaining what it's for please?